debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -687,6 +697,13 @@ public boolean isContinuousMining() {
         return getConfig().getBoolean(CONTINUOUS_MINE);
     }
 
+    /**
+     * @return the number of days the archive logs are scanned for log mining
+     */
+    public long getLogMiningArchiveLogDays() {
```

Comment:
Let's expose `Duration` here for the sake of consistency with other usages elsewhere. That said, is days a sufficient granularity?

Reply:
We can certainly use `Duration` here, that shouldn't be an issue.  As for granularity, yes because that's the lowest granularity that Oracle maintains the entries in the table, the column is of type `DATE` only.